### ISSUE TITLE
Careers page is broken

### ISSUE THREAD

**Issue by leoarnold** [mastodon/joinmastodon#428]
I don't know if you currently have job offerings - but if so, they cannot be listed on your homepage due to

```shell
$ curl -i https://api.joinmastodon.org/jobs
HTTP/2 401 
# ...
```

Technically not an issue with this repository, but I was unable to find a repo for the Rails app behind `api.joinmastodon.org`.

![Screenshot_2023-09-22_17-01-29](https://github.com/mastodon/joinmastodon/assets/4201235/d52e5a5b-945d-4946-b28a-6aac07d82315)

**Comment by andypiper** [comment]
Yikes, thanks for spotting this. I'll look into what could be happening here.

(for completeness, no roles currently to list, but good to know about this issue)

**Comment by lamemakes** [comment]
This is actually a longish standing issue - previously this would cause a client-side crash which was fixed in #358. Cheers!

**Comment by fabioselau077** [comment]
I got the same error, I tried to contribute to the adjustment but without success, I was a little lost.
In the code it sets the apiBase to https://api.joinmastodon.org/ directly, how do I test this API locally? I tried to access it in different ways and I couldn't.

**Comment by andypiper** [comment]
Fixed in #534
